lib/oli_web/controllers/project_controller.ex

```diff
@@ -102,22 +121,27 @@ defmodule OliWeb.ProjectController do
   end
 
   def insights(conn, _project_params) do
-    render conn, "insights.html", breadcrumbs: [Breadcrumb.new(%{full_title: "Insights"})], active: :insights
+    render(conn, "insights.html",
```

Comment:
There seems to be a lot of "auto formatted" changes here. We should synchronize across all of the engineering team the standard that we are formatting to and the tools that we are using to do that formatting, otherwise we will face a lot of these types of diffs during PR review.